go.mod

```diff
@@ -26,11 +27,13 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
-	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/iavl v0.14.0
-	github.com/tendermint/tendermint v0.33.7
+	github.com/tendermint/go-amino v0.15.0
+	github.com/tendermint/iavl v0.14.1
+	github.com/tendermint/tendermint v0.33.8
 	github.com/tendermint/tm-db v0.5.1
-	google.golang.org/grpc v1.30.0 // indirect
+	golang.org/x/net v0.0.0-20201010224723-4f7140c49acb // indirect
+	golang.org/x/sys v0.0.0-20201013132646-2da7054afaeb // indirect
+	google.golang.org/protobuf v1.25.0 // indirect
```

Comment:
I wonder what is it that is introducing these indirect dependencies? @ethanfrey

Reply:
@RiccardoM please run `go mod tidy`

Reply:
@alessio I run it before every push, so the last `go.mod` is updated. I've run it locally and it doesn't change

Reply:
Fair enough